### PR TITLE
auto formatted

### PR DESCRIPTION
**Pull request by mysterywolf**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Checkpoints
- [x] Follow the [styling guide](https://github.com/lvgl/lvgl/blob/master/docs/CODING_STYLE.md)
- [ ] Update CHANGELOG.md
- [ ] Update the documentation

### CODE REVIEW

**Comment by mysterywolf** [comment]
@xiaoxiang781216  Could you review again? Thanks.

File: src/widgets/lv_label.c [hunk 1]
```diff
@@ -1234,3 +1234,4 @@ static void set_ofs_y_anim(void * obj, int32_t v)
 
 
 #endif
+f
```
Comment (xiaoxiang781216) [inline src/widgets/lv_label.c]: please squach the second patch instead, you can:
```
git rebase --interactive HEAD~2
change pick to squach
modify the comment
git push origin -f master
```

File: src/lv_conf_internal.h [hunk 2]
```diff
@@ -29,7 +29,7 @@
 
 /*If lv_conf.h is not skipped include it*/
 #if !defined(LV_CONF_SKIP)
-#  if defined(LV_CONF_PATH)											/*If there is a path defined for lv_conf.h use it*/
+#  if defined(LV_CONF_PATH)                                         /*If there is a path defined for lv_conf.h use it*/
```
Comment (xiaoxiang781216) [inline src/lv_conf_internal.h]: src/lv_conf_internal.h is automatically generated too. Please modify https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_checker.py instead.

**Comment by xiaoxiang781216** [comment]
BTW, it will be good if you can create a new patch to incoperate your check into:
https://github.com/lvgl/lvgl/blob/master/scripts/code-format.sh

**Comment by xiaoxiang781216** [comment]
@mysterywolf please squash the pathset into one, thanks.

**Comment by embeddedt** [comment]
@xiaoxiang781216 GitHub allows us to squash merge so I think @mysterywolf shouldn't need to squash on their end.

**Comment by xiaoxiang781216** [comment]
Ok, I help you squash the patch, it looks good now.

**Review by xiaoxiang781216 — APPROVED** [review]

**Comment by mysterywolf** [comment]
Thank you

**Pull request merged**